test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] You should add this file to your `.gitignore` file. Take a look at this requirement:

![image](https://user-images.githubusercontent.com/27709832/139601220-6370c016-66ab-4adf-b499-ff58c202cc3f.png)